src/test/java/com/kainos/ea/backend/controllers/JobFamilyControllerTest.java

```diff
@@ -26,4 +39,30 @@ public void when_QueryingDataJobs_expect_ServiceCalledPassback() {
         Mockito.verify(jobFamilyService).getJobFamiliesByCapabilityName("");
         assertEquals(jobFamily, results);
     }
+
+    @Test
+    public void when_QueryingUpdateJobFamilyName_expect_ServiceCalledPassback() {
+        jobFamilyService.updateJobFamilyName("People Support", "New name");
+
+        Mockito.verify(jobFamilyService).updateJobFamilyName("People Support", "New name");
+    }
+
+    @Test
+    public void when_QueryingUpdateJobFamilyNameWithNonexistentJobFamily_expect_ServiceThrowsException() throws NoSuchElementException {
+        Mockito
```

Comment:
Unusual spacing here, normally you would do...
```
Mockito.doThrow(NoSuchElementException.class)
    .when(jobFamilyService).updateJobFamilyName("Unknown", "New name");
```
or something similar. Not a huge concern.

Reply:
Was wandering what would be the correct way, thanks for pointing it out

Reply:
If you want additional reading on some styling conventions around line wrapping
https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html
https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping